packages/odie-client/src/utils/user-provided-enough-information.ts

```diff
@@ -0,0 +1,22 @@
+import type { Message } from '../types';
```

Comment:
Using categories is a smart way to work around current tagging in the WP.com chat assistant. I think this is ok to merge so that we improve the user experience asap. But we should follow this up with further improvements to make sure we don't break the frontend by changing category names. Could you create a follow-up task for this in Linear and aim to address it next week? As far as I can see, it will require a change to the assistant yaml first.

Reply:
Yes, I agree, we should address this in ai-services